### PR TITLE
fix: fix mounted

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#58

### CODE REVIEW

**Review by qpitlove — APPROVED** [review]

**Review by malangfox — APPROVED** [review]
LGTM

File: src/Grid.ts [hunk 1]
```diff
@@ -316,7 +316,7 @@ abstract class Grid<Options extends GridOptions = GridOptions> extends Component
     // Grid: renderItems => checkReady => readyItems => applyGrid
     const items = this.items;
     const updated = items.filter((item) => item.element?.parentNode && item.updateState !== UPDATE_STATE.UPDATED);
-    const mounted: GridItem[] = updated.filter((item) => item.mountState !== MOUNT_STATE.MOUNTED);
+    const mounted = items.filter((item) => item.element?.parentNode && item.mountState !== MOUNT_STATE.MOUNTED);
```
Comment (WoodNeck) [inline src/Grid.ts:319]: I get the point, but these items look very confusing though.
they're not updated(`item.updateState !== UPDATE_STATE.UPDATED`) and mounted(`item.mountState !== MOUNT_STATE.MOUNTED`) but named as `updated` and `mounted`

**Review by WoodNeck — APPROVED** [review]
LGTM

File: src/Grid.ts (hunk 1 above)
Comment (daybrush) [reply src/Grid.ts:319]: I know what you mean.

It's not done, it's going to be, and it's also the name of the event property.

**Pull request merged**